server/organizations_test.go

```diff
@@ -223,16 +228,84 @@ func TestService_UpdateOrganization(t *testing.T) {
 					},
 					GetF: func(ctx context.Context, q chronograf.OrganizationQuery) (*chronograf.Organization, error) {
 						return &chronograf.Organization{
-							ID:   1337,
-							Name: "The Good Place",
+							ID:          1337,
+							Name:        "The Good Place",
+							DefaultRole: roles.ViewerRoleName,
+							Public:      true,
+						}, nil
+					},
+				},
+			},
+			id:              "1337",
+			wantStatus:      http.StatusOK,
+			wantContentType: "application/json",
+			wantBody:        `{"id":"1337","name":"The Bad Place","defaultRole":"viewer","links":{"self":"/chronograf/v1/organizations/1337"},"public":true}`,
+		},
+		{
+			name: "Update Organization public",
```

Comment:
could you update the default org specifically, rather than an arbitrary org, since we only plan currently to allow the default org to be toggled in terms of `public`?